src/__tests__/lib/containers/QuerySortSelector.test.tsx

```diff
@@ -0,0 +1,100 @@
+import { act, render, screen, waitFor, fireEvent } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import React from 'react'
+import { SynapseConstants } from '../../../lib/utils'
+import QuerySortSelector, {
+  QuerySortSelectorProps,
+} from '../../../lib/containers/QuerySortSelector'
+import {
+  QueryVisualizationContextProvider,
+  QueryVisualizationContextType,
+} from '../../../lib/containers/QueryVisualizationWrapper'
+import {
+  QueryContextProvider,
+  QueryContextType,
+} from '../../../lib/containers/QueryContext'
+import { createWrapper } from '../../../lib/testutils/TestingLibraryUtils'
+import {
+  QueryBundleRequest,
+  QueryResultBundle,
+} from '../../../lib/utils/synapseTypes'
+import syn16787123Json from '../../../mocks/query/syn16787123'
+
+const renderComponent = (
+  props: QuerySortSelectorProps,
+  queryContext: Partial<QueryContextType>,
+) => {
+  const defaultQueryVisualizationContext: Partial<QueryVisualizationContextType> =
+    {}
+
+  return render(
+    <QueryContextProvider queryContext={queryContext}>
+      <QueryVisualizationContextProvider
+        queryVisualizationContext={defaultQueryVisualizationContext}
+      >
+        <QuerySortSelector {...props} />
+      </QueryVisualizationContextProvider>
+    </QueryContextProvider>,
+    {
+      wrapper: createWrapper(),
+    },
+  )
+}
+
+describe('QuerySortSelector tests', () => {
```

Comment:
Can we also test flipping the sort direction when re-selecting it (if you decide to keep that functionality?)

Reply:
Is there a way for me to interrogate the query request object (to verify the SortItems)?

Reply:
Like this?
```
expect(executeQueryRequest).toHaveBeenCalledWith(expect.objectContaining({
	query: expect.objectContaining({
		sort: {
			// expected sort column and direction here
		}
	})
}))
```